netctl/netctl.go

```diff
@@ -401,3 +402,14 @@ func dumpList(ctx *cli.Context, list []map[string]interface{}) {
 	os.Stdout.Write(content)
 	os.Stdout.WriteString("\n")
 }
+
+func showVersion(ctx *cli.Context) {
```

Comment:
@jojimt, @mapuri - added versions for k8splugin, and netctl independently; also the output of netctl version shows the client/server(netmaster) versions together.

the pending item from above discussion is really netplugin version; I suggest we skip that for this PR until we start putting versions for APIs